sdk/communication/azure-communication-callingserver/src/main/java/com/azure/communication/callingserver/CallConnection.java

```diff
@@ -3,221 +3,258 @@
+     * Hangup a call.
      *
-     * @param audioFileUri The media resource uri of the play audio request. Currently only Wave file (.wav) format
-     *                     audio prompts are supported. More specifically, the audio content in the wave file must
-     *                     be mono (single-channel), 16-bit samples with a 16,000 (16KHz) sampling rate.
-     * @param playAudioOptions Options for play audio.
      * @throws CallingServerErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return Response payload for play audio operation.
+     * @return Response for a successful hangup request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public PlayAudioResult playAudio(String audioFileUri, PlayAudioOptions playAudioOptions) {
-        return callConnectionAsync.playAudioInternal(audioFileUri, playAudioOptions).block();
+    public Void hangup() {
+        return callConnectionAsync.hangup().block();
     }
 
     /**
-     * Play audio in the call.
+     * Hangup a call.
      *
-     * @param audioFileUri The media resource uri of the play audio request. Currently only Wave file (.wav) format
-     *                     audio prompts are supported. More specifically, the audio content in the wave file must
-     *                     be mono (single-channel), 16-bit samples with a 16,000 (16KHz) sampling rate.
-     * @param playAudioOptions Options for play audio.
      * @param context A {@link Context} representing the request context.
      * @throws CallingServerErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
-     * @return Response payload for play audio operation.
+     * @return Response for a successful hangup request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public Response<PlayAudioResult> playAudioWithResponse(
-        String audioFileUri,
-        PlayAudioOptions playAudioOptions,
-        final Context context) {
-        return callConnectionAsync
-            .playAudioWithResponseInternal(audioFileUri, playAudioOptions, context)
-            .block();
+    public Response<Void> hangupWithResponse(Context context) {
+        return callConnectionAsync.hangupWithResponseInternal(context).block();
     }
 
+
     /**
-     * Disconnect from a call.
+     * Terminates the conversation for all participants in the call.
      *
      * @throws CallingServerErrorException thrown if the request is rejected by server.
      * @throws RuntimeException all other wrapped checked exceptions if the request fails to be sent.
+     * @return Response for a successful call termination request.
      */
     @ServiceMethod(returns = ReturnType.SINGLE)
-    public void hangup() {
-        callConnectionAsync.hangup().block();
+    public Void terminateCall() {
```

Comment:
Terminate call to be consolidated with HangUp.
HangUp will take bool input forEveryone
if false, it is hangup
if true, it is terminate

Reply:
fixed